torch/ao/quantization/_dbr/module_swap_utils.py

```diff
@@ -23,16 +27,21 @@ def _swap_child_modules(
         # both fused modules and observed custom modules are
         # swapped as one unit
         if not isinstance(mod, _FusedModule):
-            _swap_child_modules(mod, static_mappings)
+            _swap_child_modules(mod, static_mappings, dynamic_mappings)
 
         qconfig = getattr(mod, 'qconfig', None)
         if not qconfig:
             continue
         activation_int8_quantized = activation_is_int8_quantized(qconfig)
+        op_int8_dynamically_quantized = op_is_int8_dynamically_quantized(qconfig)
```

Comment:
might be good to move op_is_int8_dynamically_quantized function to https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/utils.py so that other tools can use it as well